modules/testsuite/cxf-tests/src/test/scripts/jbws-testsuite-default-elytron.groovy

```diff
@@ -11,6 +11,120 @@ if (!session.userProperties['enableServerLoggingToConsole'] && !project.properti
+authenticationJaspiClient.appendNode('auth-module', ['code':'org.jboss.wsf.stack.cxf.jaspi.client.module.SOAPClientAuthModule','login-module-stack-ref':'jaas-lm-stack'])
+
+/** -------------------- legacy security subsystem ------------------------
+ * NOTE: that subsystem will be removed in future versions.
+ * Add a security-domain block for test PicketboxUsernameAuthorizationDigestEjbTestCase
+ *
+ * <security-domain name="PicketboxJBossWSDigest" cache-type="default">
+ *   <authentication>
+ *     <login-module code="UsersRoles" flag="required">
+ *       <module-option name="hashUserPassword" value="false"/>
+ *       <module-option name="hashCharset" value="UTF-8"/>
+ *       <module-option name="usersProperties" value="/mnt/ssd/jbossws/stack/cxf/trunk/modules/testsuite/cxf-tests/target/test-resources/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-users.properties"/>
+ *       <module-option name="hashAlgorithm" value="SHA"/>
+ *       <module-option name="unauthenticatedIdentity" value="anonymous"/>
+ *       <module-option name="hashEncoding" value="BASE64"/>
+ *       <module-option name="rolesProperties" value="/mnt/ssd/jbossws/stack/cxf/trunk/modules/testsuite/cxf-tests/target/test-resources/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-roles.properties"/>
+ *       <module-option name="storeDigestCallback" value="org.jboss.wsf.stack.cxf.security.authentication.callback.UsernameTokenCallback"/>
+ *       <module-option name="hashStorePassword" value="true"/>
+ *     </login-module>
+ *   </authentication>
+ * </security-domain>
+ *
+ */
+def securityDomainDigest = securityDomainsLegacy.appendNode('security-domain', ['name':'PicketboxJBossWSDigest','cache-type':'default'])
+def authenticationDigest = securityDomainDigest.appendNode('authentication')
+def loginModuleDigest = authenticationDigest.appendNode('login-module', ['code':'UsersRoles','flag':'required'])
+loginModuleDigest.appendNode('module-option', ['name':'hashUserPassword','value':'false'])
+loginModuleDigest.appendNode('module-option', ['name':'hashCharset','value':'UTF-8'])
+loginModuleDigest.appendNode('module-option', ['name':'hashAlgorithm','value':'SHA'])
+loginModuleDigest.appendNode('module-option', ['name':'hashEncoding','value':'BASE64'])
+loginModuleDigest.appendNode('module-option', ['name':'storeDigestCallback','value':'org.jboss.wsf.stack.cxf.security.authentication.callback.UsernameTokenCallback'])
+loginModuleDigest.appendNode('module-option', ['name':'hashStorePassword','value':'true'])
+loginModuleDigest.appendNode('module-option', ['name':'unauthenticatedIdentity','value':'anonymous'])
+loginModuleDigest.appendNode('module-option', ['name':'usersProperties','value':testResourcesDir + '/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-users.properties'])
+loginModuleDigest.appendNode('module-option', ['name':'rolesProperties','value':testResourcesDir + '/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-roles.properties'])
+
+
+
+
+
```

Comment:
We'd better add this in a groovy script dedicated to configure legacy subsystem to run the tests instead of get all these things mixed in both legacy and elytron configuration.

Reply:
changed